Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.py

```diff
@@ -430,6 +447,10 @@ def test_func():
         except ValueError as e:
             return_error("Inserted time format is incorrect.\n" + str(e) + '\n' + TIME_FIELD + ' fetched: ' + hit_date)
 
+    else:
+        # check that we can reach any indexes in the supplied server URL
+        test_general_query(es)
```

Comment:
why are we doing it only in case fetch is not set ?

Reply:
If fetch is set then we are checking if we can approach the specific index for fetch - this serves the same purpose as this check so I only test this if the fetch is not set